Core/clim-core/menu.lisp

```diff
@@ -170,7 +175,10 @@ account, and create a list of menu buttons."
 	    (adopt-frame manager submenu-frame)
 	    (enable-frame submenu-frame)
 	    (with-sheet-medium (medium raised)
-	      (medium-force-output medium))))))))
+	      (medium-force-output medium))))))
+    (when frame
```

Comment:
functions earlier would signal a condition if frame is  nil, so there is no need for this test

Reply:
Okies